scripts/src/main/resources/scripts/migration

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/functions
+
+from_version=2023.02.001
+if [ -f "${DEVON_IDE_HOME}/.devon.software.version" ]  
+then
+  from_version="$(cat ${DEVON_IDE_HOME}/.devon.software.version)"
+fi
+
+MIGRATIONS_DIR="${DEVON_IDE_HOME}/scripts/migrations"
+
+if [ -d "${MIGRATIONS_DIR}" ]
+then
+  cd "${MIGRATIONS_DIR}"
+  for migrate_to_version in  $(ls | sort -n)
+  do
+    doVersionCompare "${from_version}" "${migrate_to_version}"
+    result="${?}"
+    if [ "${result}" -eq 0 ] || [ "${result}" -eq 2 ]
+    then
+      doEcho "Run migration to version ${migrate_to_version}"
+      bash "${migrate_to_version}"
+      migration_result="${?}"
+      if [ "${migration_result}" -ne 0 ]
+      then
+        doAskToContinue "The migration to version ${migrate_to_version} failed. Would you like to continue?"
+      fi
```

Comment:
if no error happened, we need to update `${DEVON_IDE_HOME}/.devon.software.version)` to `${migrate_to_version}`.

Reply:
Done.